### DISCUSSION TITLE
MaterialDesignFloatingHintTextBox's Hint Always Floating above the TextBox.

### DISCUSSION THREAD

**Discussion by MeshkaM** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3401] [Q&A] [answered]
How can i make the MaterialDesignFloatingHintTextBox's Hint to stay to always float above the TextBox, regardless of whether the TextBox is empty or not.

**Comment by Keboo** [comment, marked as answer]
At present there is not a simple way to achieve this without manipulating the content. This is because the internal `SmartHint` control uses the "empty" content to choose if it will be in the floating position (see: https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/59375fff6485380f70ac8cc3b002f8263d8454ff/MaterialDesignThemes.Wpf/SmartHint.cs#L206-L238).

If you wanted to role your own template you _could_ use the `SmartHint` control inside of the template, create your `IHintProxy` that always returns `true` for `IsEmpty()`. But that is likely more work than simply placing an additional `TextBlock` positioned where you want it.
